selfdrive/camerad/cameras/camera_common.cc

```diff
@@ -361,6 +364,19 @@ void *processing_thread(MultiCameraState *cameras, const char *tname,
 
     callback(cameras, cs, cnt);
 
+#if defined(QCOM) || defined(QCOM2)
```

Comment:
if we start putting ifdefs like this everywhere, camera_common will just become what main.cc used to be

Reply:
yes you're right,I'll reconsider it.

Reply:
yes you're right,I'll reconsider it.